p2p/les.py

```diff
@@ -86,7 +95,7 @@ def decode_payload(self, rlp_data: bytes) -> Generator[Tuple[str, Any], None, No
             else:
                 yield key, value
 
-    def encode_payload(self, data):
+    def encode_payload(self, data: Dict[str, bytes]) -> bytes:  # type: ignore
```

Comment:
Why do we need the `type: ignore` comment here?

Reply:
It doesn't conform to the signature of the super type.

We could also do this:
```
    def encode_payload(self, data: Union[_DecodedMsgType, sedes.CountableList]) -> bytes:
        response = [
            (key, self.items_sedes[key].serialize(value))
            for key, value
            in sorted(data.items())  # type: ignore
        ]
        return super().encode_payload(response)
```
So, we conform to the interface but we then ignore that `data.items()` is actually invalid because `items()` doesn't exist on a Tuple.